app/data_providers/slate_providers/for_you_slate_provider.py

```diff
@@ -49,7 +62,19 @@ async def rank_corpus_items(
         assert preferred_topics is not None
         assert user_impression_capped_list is not None
 
-        random.shuffle(items)
+        if kwargs.get('enable_thompson_sampling'):
+            metrics = await self.corpus_engagement_provider.get(
+                self.recommendation_surface_id, self.configuration_id, items)
+
+            items = thompson_sampling(
+                recs=items,
+                metrics=metrics,
+                trailing_period=14,  # A long period might work better given that some topics get few impressions
+                default_alpha_prior=10,  # beta * P95 item CTR for this slate (1.6%)
+                default_beta_prior=600)  # 5% of average daily item impressions for this slate
```

Comment:
double parameters here to account for somewhat longer exploration period until prior gets absorbed in clickdata